pages/linux/vso.md

```diff
@@ -1,36 +1,36 @@
 # vso
 
-> Utility to perform maintenance tasks on Vanilla OS.
+> Package manager, system updater and a task automator for Vanilla OS.
 > More information: <https://github.com/Vanilla-OS/vanilla-system-operator>.
 
-- Trigger a system update immediately:
+- Check for system updates to the host system:
 
-`sudo vso trigger-update --now`
+`vso sys-upgrade check`
 
-- Check for package updates and list them:
+- Upgrade the host system now:
 
-`sudo vso update-check`
+`vso sys-upgrade upgrade --now`
 
-- Create an automated task upon an application's launch:
+- Initialize the subsystem:
```

Comment:
(copy-paste from the readme, "initialize the subsystem" sounds very cryptic to me ngl)

```suggestion
- Initialize the Pico subsystem (used for package management):
```

Reply:
Thanks for the keen observation, I wrote this a while ago and I missed updating this line when creating the PR. Will apply this now.

> "initialize the subsystem" sounds very cryptic to me ngl

Agreed, the same suggestion was brought up in https://github.com/Vanilla-OS/vanilla-system-operator/pull/53#discussion_r1302855717 by a fellow docs team member and I changed it in docs after it.